index.js

```diff
@@ -11,9 +11,12 @@ function enableDestroy(server) {
     });
   });
 
-  server.destroy = function(cb) {
+  server.destroy = function(cb, time) {
```

Comment:
This violates the node convention for having the callback be the last argument. You could do this:

```js
server.destroy = function(options, cb) {
  if (!cb) {
     cb = options;
     options  = {};
  }

  var time = options.time || 0;

  // rest of code
```